packages/cbioportal-ts-api-client/src/generated/CBioPortalAPI-docs.json

```diff
@@ -5142,6 +5222,30 @@
       },
       "title": "AlleleSpecificCopyNumber"
     },
+    "AndedPatientTreatmentFilters": {
+      "type": "object",
+      "properties": {
+        "filters": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/OredPatientTreatmentFilters"
```

Comment:
is it correct that AndedPatientTreatmentFilters is an array of OredPatientTreatmentFilters? 

Would it be possible to have just PatientTreatmentFilters type with a property of value "and|or" ?

Reply:
I don't think so, no. Java does not have sum types.